app/views/friendships/_friends_list.html.erb

```diff
@@ -12,8 +12,20 @@
 		<% else %>
 		  <td ><strong>Friends Since: <%= time_ago_in_words(friendship.request_responded_at).capitalize %> ago</strong></td>
 		<% end %>
-	  <td class="champ-img" rowspan="3" scope="rowgroup"><span class="imgWrap"><%= link_to image_tag("icons/AhriSquare.png"), profile_path(friendship.friend.id)  %></span>
-    </td>
+	  <td rowspan="3" scope="rowgroup">
+      <div class="champ-img">
+        <span class="imgWrap"><%= link_to image_tag("icons/AhriSquare.png"), profile_path(friendship.friend.id)  %></span>
+        <div class="status-icon">
+	        <% if friendship.friend.logged_in == true %>
```

Comment:
You don't need to compare this with the boolean.  `if friendship.friend.logged_in` will do the same thing